week-4/Homework/mandatory/1-food.js

```diff
@@ -26,22 +27,29 @@ console.log("I can buy my favorite food from " + getTakeAwayAddress(myFavFood));
 // and returns true if it's vegetarian, or false if not
 
 function isVeggie(food) {
+  return food.isVeggie;
   // write code here
 }
 
-console.log("Is my favorite vegetarian? " + isVeggie(food));
+console.log("Is my favorite vegetarian? " + isVeggie(myFavFood));
 
 // 3. Declare a function "isLowCalorie" that accepts "food" as a parameter
 // and returns true if it has less than 600 calories, or false otherwise
 // write your own console.log() that calls the function to test it, as in the examples above!
 // no example code is provided here!
 
+function isLowCalorie(food) {
+  return food.caloriesPerPortion < 600;
+};
+
+console.log("Is the food low calorie? " + isLowCalorie(myFavFood));
 //---------------------------------------
 
 // 3. Declare a function "isSafeForNutAllergies" that accepts "food" as a parameter
 // and returns false if it found the word "sesame" in the ingredients, or true otherwise
 
 function isSafeForNutAllergies(food) {
+  return !food.ingredients.includes("sesame seeds");
   //write code here
```

Comment:
You need to check for the word "sesame" in the ingredients so it's a bit more than checking if the `food.ingredients` array includes sesame seeds. You need to use an array function that will iterate (loop) through the ingredients and then check whether each string contains that function. I would suggest that you will need to use an array function, a function of your own that you use on each element and then a function of string to check what each one includes. From this you should be able to work out whether it's true or not that there is at least one ingredient with the word sesame in it.